main.go

```diff
@@ -21,6 +21,7 @@ import (
 const (
 	cacheInfoFilePath = "/tmp/cache-info.json"
```

Comment:
NOTE: oops, /tmp is a different partition then the os default temp dir, so creating and using temp files from this dir might will be a lot slower

Reply:
What do you suggest?